database/migrations/legacy/2012_10_03_133633_create_base_tables.php

```diff
@@ -375,6 +376,12 @@ public function up()
             // UPDATE `GameData` SET `Created` = NULL WHERE `Created` IS NOT NULL;
         }
 
+        if (!Schema::hasColumns('GameData', ['GuideURL'])) {
+            Schema::table('GameData', function (Blueprint $table) {
+                $table->string('GuideURL')->nullable();
+            });
+        }
```

Comment:
This should be its own step, or databases that have already run this step will not apply the change. I don't think we have any for adding a column yet, but here's one for adding an index: https://github.com/RetroAchievements/RAWeb/blob/master/database/migrations/legacy/2022_10_01_000000_update_ticket_index.php.

Basically, you'll end up with these five lines in your `up()` and a `dropColumn` call in your `down()` function.

Running `php artisan migrate` should add the column and `php artisan migrate:rollback` should remove it.

Reply:
Cool, that makes perfect sense. I was wondering about this. Will revise later today.

Reply:
All set